services/src/main/java/org/apache/druid/cli/CliHadoopIndexer.java

```diff
@@ -84,8 +84,8 @@ public void run()
         extensionURLs.addAll(Arrays.asList(((URLClassLoader) extensionLoader).getURLs()));
       }
 
-      final List<URL> nonHadoopURLs = new ArrayList<>();
-      nonHadoopURLs.addAll(Arrays.asList(((URLClassLoader) CliHadoopIndexer.class.getClassLoader()).getURLs()));
+      final List<URL> nonHadoopURLs = new ArrayList<>(Arrays.asList(((URLClassLoader) CliHadoopIndexer.class.getClassLoader())
```

Comment:
The `new ArrrayList<>()` seems unnecessary?

Reply:
removed